lib/fluent/test/base.rb

```diff
@@ -37,7 +37,7 @@ def initialize(klass, &block)
         else
           @instance = klass
         end
-        @instance.router = Engine.root_agent.event_router
+        @instance.router = Engine.root_agent.event_router if @instance.respond_to?(:router=)
```

Comment:
I removed it.
https://github.com/fluent/fluentd/pull/1154/commits/3877b3ac9ae200ae6a55725d6580bb806c380cd4#diff-11193d96f15fd472415e15b1c787938eL40